packages/flutter/lib/src/rendering/editable.dart

```diff
@@ -1912,6 +1912,38 @@ class RenderEditable extends RenderBox with RelayoutWhenSystemFontsChangeMixin {
     // If text is obscured, the entire sentence should be treated as one word.
     if (obscureText) {
       return TextSelection(baseOffset: 0, extentOffset: _plainText.length);
+    // If the word is a space, on iOS try to select the previous word instead.
+    } else if (text?.text != null
+        && _isWhitespace(text!.text!.codeUnitAt(position.offset))
+        && position.offset > 0) {
+      assert(defaultTargetPlatform != null);
+      switch (defaultTargetPlatform) {
+        case TargetPlatform.iOS:
+          int startIndex = position.offset - 1;
+          while (startIndex > 0 && _isWhitespace(text!.text!.codeUnitAt(startIndex))) {
```

Comment:
nit: it should skip line breaks and `\u200e` `\u200f` as well

Reply:
👍  Added \u200e and \u200f.

It seems to work currently for skipping over lines, is there a specific line break character you mean that is not covered in _isWhitespace?